src/modes/simple_select.js

```diff
@@ -26,6 +27,32 @@ module.exports = function(ctx, options = {}) {
     });
   };
 
+  const fireActionable = actionables(ctx, () => {
+    var selectedFeatures = ctx.store.getSelected();
+
+    var multiFeatures = selectedFeatures.filter(
+      feature => feature instanceof MultiFeature
+    );
+
+    var combine = false;
+
+    if (selectedFeatures.length) {
+      var featureType = selectedFeatures[0].type.replace('Multi','');
+      selectedFeatures.forEach(feature => {
+        if(feature.type.replace('Multi','') === featureType) {
+          combine = true;
```

Comment:
We should flip this logic to start as true and move to false if something that doesn't match the `featureType` is found.